mautic/mautic
# [UI] Styling for HTML keyboard tags

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

This PR styles kbd tags to look better.




| Before                                 | After
| -------------------------------------- | ---
|    ![image](https://github.com/mautic/mautic/assets/116097999/4b11269b-540e-416b-969c-ba7a7752b505)                                    |  ![image](https://github.com/mautic/mautic/assets/116097999/fd19a381-e9b7-45cb-8bf1-33dc5d54d44d)




---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Type in your keyboard `Shift + ?` to open the keyboard shortcuts menu
3. See the style
